tests/calculator-homework.spec.js

```diff
@@ -0,0 +1,116 @@
+const { test, expect } = require('@playwright/test');
+const {calcStartPage} = require('../pages/calcStartPage');
+
+test.describe('Calculator test suite', () => {
+  let page;
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+      startPage = new calcStartPage(page);
+      
+  });
+  test.beforeEach(async () => {
+    await startPage.goto();
+  });
+
+const buildOptions = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+
+    buildOptions.forEach(buildOption => {
+    test(`Check if Build Option ${buildOption} has a Second Number field`, async () => {
+        await page.selectOption('select[name="selectBuild"]', (buildOption));
+        const isFieldVisible = await page.isVisible('#number2Field');
+        expect(isFieldVisible).toBe(true)
+    });
+  }); 
+
+
+  buildOptions.forEach(buildOption => {
+    test(`Check if Add function provides the correct answers with Build Option ${buildOption}`, async () => {
+        await page.selectOption('select[name="selectBuild"]', (buildOption));
+        await page.click('input[name="number1"]');
```

Comment:
Is it really required to click the input? Maybe fill function bellow already does it itself.